src/components/Network/index.js

```diff
@@ -71,15 +72,29 @@ class Network extends Component {
     this._isMount = false;
   }
 
+  getUUIDS = data => {
+    let uuids = [];
+
+    if (data) {
+      uuids.push(data.key);
+    }
+
+    return uuids;
+  };
+
   getData() {
     this.setState({ loading: true });
 
     getNetwork(this.props.match.params.key).then(({ data }) => {
       // If user lives the page, request will return result anyway and tries to set in to a state
       // which will cause an error
       if (this._isMount) {
+        // Taken an array of UIDs to check user permissions
+        const uuids = this.getUUIDS(data);
```

Comment:
This seem unnecessary. We already have the key. Why do a lookup to the api to get the key again, and then have a function to add that. Just `const uuids = this.props.match.params.key` would do wouldn't it?

Reply:
indeed, I've replaced that